packages/compass-preferences-model/src/feature-flags.ts

```diff
@@ -192,7 +192,7 @@ export const featureFlags: Required<{
   },
 
   enableAutomaticRelationshipInference: {
-    stage: 'development',
+    stage: 'released',
```

Comment:
do we remove FF's once they've been fully released?

Reply:
Sooo... we should be cleaning them up over time, but practically we always forget to schedule this work, so a lot of very old flags are hanging out in the codebase (tell me if this sounds familiar @johnjackweir 😄) and as Anna correctly noticed there's not really much harm in that and so very little pressure for cleaning up

The main reason we have a dedicated `released` state is to be able to easy roll back in the first few weeks of the feature being available by just flipping the value instead of, for example, reverting the patches that would remove the flag from the code completely. But after this "trial" period we should be cleaning those up, in theory